src/pancake.md

```diff
@@ -1,25 +1,33 @@
-# Pancake
+# Pancakes
 
 ![Pancakes made at home](pix/pancake.webp)
 
 Here is a simple Pancake recipe
 
+- ⏲️ Prep time: 20 min
+- 🍳 Cook time: 10 min
+- 🍽️ Servings: 1
+
 ## Ingredients
-- 200g wheat flour
+- 200g wheat flour / 100g glutenfree flour
 - 200 ml milk
-- 2 Eggs
-- 2 teaspoon sugar
-- 1 teaspoon baking soda
+- 2 eggs
+- 2 tsp sugar
+- 1 tsp baking soda
 - a little bit of salt
 
 ## Directions
-1. combine all ingredients all in a bowl and stir
-2. preheat a pan and add some oil
-3. Pour or scoop the batter onto the pan
-4. Brown on both sides
+1. Mix all ingredients in a bowl
+2. Let the dought rest for ~15 mins
+3. Heat a pan with a bit butter or oil
+4. For each pancake spread a ladle of dought in the pan and bake them at lower medium temperature until they are brown on both sides
 
-## Contribution
+## Served with
+- Syrup
+- Cinnamon and sugar
 
-BeFe
+## Contribution
+- BeFe (original)
+- Lxgr (gluten-free) - [github](https://github.com/lxgr-linux)
 
-;tags: quick breakfast sweet pancake cheesefare
+;tags: quick basic breakfast sweet pancake cheesefare
```

Comment:
```suggestion
;tags: quick breakfast sweet pancake cheesefare
```
Not sure if basic should be added. It's really meant for recipes that are not meant to be done alone as a dish, but pancakes on their own (except for the addition of syrup or something) can pretty much be their own thing.

Reply:
Also, 6 tags is too many